website/docs/r/mobile_network_sim.html.markdown

```diff
@@ -0,0 +1,134 @@
+    slice_id                 = azurerm_mobile_network_slice.test.id
+    static_ipv4_address      = "2.4.0.1"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Mobile Network Sim. Changing this forces a new Mobile Network Sim to be created.
+
+* `mobile_network_sim_group_id` - (Required) The ID of the Mobile Network which the Mobile Network Sim belongs to.
+
+* `authentication_key` - (Required) The Ki value for the SIM.
+
+* `international_mobile_subscriber_identity` - (Required) The international mobile subscriber identity (IMSI) for the SIM. Changing this forces a new Mobile Network Sim to be created.
+
+* `integrated_circuit_card_identifier` - (Required) The integrated circuit card ID (ICCID) for the SIM. Changing this forces a new Mobile Network Sim to be created.
+
+* `operator_key_code` - (Required) The Opc value for the SIM.
+
+* `device_type` - (Optional) An optional free-form text field that can be used to record the device type this SIM is associated with, for example `Video camera`. The Azure portal allows SIMs to be grouped and filtered based on this value.
+
+* `sim_policy_id` - (Optional) The ID of SIM policy used by this SIM.
+
+* `static_ip_configuration` - (Optional) A `static_ip_configuration` block as defined below.
+
+---
+
+A `static_ip_configuration` block supports the following:
+
+* `attached_data_network_id` - (Required) The ID of attached data network on which the static IP address will be used. The combination of attached data network and slice defines the network scope of the IP address.
+
+* `slice` - (Required) The ID of network slice on which the static IP address will be used. The combination of attached data network and slice defines the network scope of the IP address.
+
+* `static_ipv4_address` - (Optional) The IPv4 address assigned to the SIM at this network scope. This address must be in the userEquipmentStaticAddressPoolPrefix defined in the attached data network.
+
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Mobile Network Sim.
+
+* `sim_state` - The state of the SIM resource.
+
+* `vendor_key_fingerprint` - The public key fingerprint of the SIM vendor who provided this SIM, if any.
+
+* `vendor_name` - The name of the SIM vendor who provided this SIM, if any.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 90 minutes) Used when creating the Mobile Network Sim.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Mobile Network Sim.
+* `update` - (Defaults to 90 minutes) Used when updating the Mobile Network Sim.
+* `delete` - (Defaults to 90 minutes) Used when deleting the Mobile Network Sim.
```

Comment:
It looks like the create, update and delete are 180 minutes in the resource rather than 90 minutes. Is there a reason why this resource has longer timeouts than normal?

Reply:
Have updated the code to keep consistent with document.